### ISSUE TITLE
Remove scoped NPM package and use a non-scoped one?

### ISSUE THREAD

**Issue by inf3rno** [inf3rno/e2e#113]
I misunderstood the scoped package feature of npm. I just checked and I cannot install the lib with `npm install e2e-testing`. I have to use `npm install @inf3rno/e2e-testing` instead. Well that sucks. The old NPM was 10 times better. I did not have to do 1000 unecessary steps to publish a single repo. At least this scoping more or less solves the problem, that all the names are taken by inactive people. I'll think about renaming the repo instead to `@inf3rno/e2e` and keep this scoping thing.

**Comment by inf3rno** [comment]
I am not sure. I mean it is easy to remember to the e2e-testing name, because

> what do I need?
> end to end testing
> let's install e2e-testing then

the scoped version is a little bit cryptic, because you need to remember the username and the syntax of the scoped package names. So it will be copy-paste only.

**Comment by inf3rno** [comment]
I can no longer remove the e2e-testing from npm, but I can ask them in email to do so. I can still publish a new one with `@inf3rno/e2e` name. I'll move all of my project under this scope on the long run, and send an email with a long list to remove my old packages. Another option to make them deprecated for a year and remove them only after that.

**Comment by inf3rno** [comment]
I'll rename this repo too.

**Comment by inf3rno** [comment]
The library does not have a problem, this is a documentation issue. The package name documented is different than what I published to NPM. I think I'll fix the docs instead. I decided yesterday to use the e2e name instead, but now I am not certain. I am ok with the e2e-testing name too. On the other hand @inf3rno/e2e is short enough. I just managed the scoped name fix on Promise.exposed too. So I can do it here too. I just need a stable version. For now I'll fix the readme and deprecate the @inf3rno/e2e-testing.
